docker/django/docker-entrypoint.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+set -e
+
+case "$1" in
+'celery')
+    exec celery \
+        --app=cl worker \
+        --loglevel=info \
+        --events \
+        --pool=prefork \
+        --hostname=prefork@%h \
+        --queues=${CELERY_QUEUES} \
```

Comment:
How certain are you that this variable will get picked up somehow?

Reply:
It basically depends on how you are currently setting these variables. One way to set them would be, depending on how the k8s stuff actually looks:

```kind: Deployment
metadata:
  name: cl-celery
spec:
  replicas: 1
  template:
    spec:
      containers:
      - name: worker
        image: freelawproject/courtlistener:latest
        args: ["celery"]
        env:
        - name: CELERY_QUEUES
          value: "<whatever>"
        
        ```
        
 But yeah, it's up to you. I think this Kubernetes manifest YAML way is probably the easiest, but I think you can also utilize a lot of other AWS secret management options that strike me as unnecessarily complicated here.